src/Files.App/ViewModels/Dialogs/BulkRenameDialogViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Windows.Storage;
+
+namespace Files.App.ViewModels.Dialogs
+{
+	public sealed class BulkRenameDialogViewModel : ObservableObject
+	{
+		private IContentPageContext context { get; } = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		// Properties
+
+		public bool IsNameValid =>
+			FilesystemHelpers.IsValidForFilename(fileName) && !fileName.Contains(".");
+
+		public bool ShowNameWarning =>
+			!string.IsNullOrEmpty(fileName) && !IsNameValid;
+
+
+		private string fileName = string.Empty;
```

Comment:
```suggestion
		private string _FileName = string.Empty;
```

Reply:
Isn't the '_' prefix against the code style rules for C#? I think it's IDE1006
https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/style-rules/naming-rules

Reply:
This is a private field. Using PascalCase is guided by our coding guideline, Section 2, Number 2.

Reply:
Oh yeah, I forgot why that seemed so familiar...